src/prisma/prisma.module.ts

```diff
@@ -0,0 +1,8 @@
+import { Module } from '@nestjs/common';
+import { PrismaService } from './prisma.service';
+
```

Comment:
add @global decorator here to import this module in every other modules without having to define the PrismaService at every module